### DISCUSSION TITLE
how to connect weed shell from inside seaweedfs docker container?

### DISCUSSION THREAD

**Discussion by sj2ch15** [seaweedfs/seaweedfs#5303] [Q&A]
I read the existing discussion and tried to configure TTL for my seaaweedfs bucket.
But When trying to access the weed shell from inside the container, while running any command I am getting an error response like this!

Can anyone pls guide here!
Assuming weed is somehow not able to communicate.

```
> s3.bucket.list
error: read buckets: getOrCreateConnection : fail to dial : failed to build resolver: passthrough: received empty target in Build()
> fs.ls
error: getOrCreateConnection : fail to dial : failed to build resolver: passthrough: received empty target in Build()
> 
```

**Comment by chrislusf** [comment]
When starting `weed shell`, how did you specify the command line options?
